src/Entity/Rdf.php

```diff
@@ -434,4 +455,25 @@ public function hasGraph($graph) {
     return $this->entityTypeManager()->getStorage($this->getEntityTypeId())->hasGraph($this, $graph);
   }
 
+  /**
+   * Returns whether the entity bundle has mapping for a certain field column.
+   *
+   * @param string $field
+   *   The field name.
+   * @param string $column
+   *   The field column. Defaults to 'value'.
+   *
+   * @return bool
+   *   True if the mapping is set, false otherwise.
+   */
+  protected function hasFieldMapping($field, $column = 'value') {
+    if (empty($this->bundle())) {
+      return FALSE;
+    }
```

Comment:
This seems strange to me, why would this function be called if the entity has no bundle? Is this perhaps being called when a new entity is being instantiated?

If not, wouldn't an exception be more appropriate? Because now it returns `FALSE` which can be interpreted as "the mapping is not set", but actually this functionality is undefined.

Reply:
I think it's mostly used when creating a new bundle or something like that from the interface.
I must say that I don't know exactly, I didn't want to change it now. Maybe @idimopoulos knows more.

Reply:
Normally an entity is created with a bundle. The property should normally be filled. I don't mind throwing an exception, I don't have a strong opinion. The check is only so that the next line's $this->bundle() does not throw an exception. Normally an entity cannot be created without a bundle so I do not know if we need it. We can try to remove it and see how are tests behave.